langstream-cli/src/main/java/ai/langstream/cli/commands/profiles/ImportProfileCmd.java

```diff
@@ -42,7 +42,8 @@ public class ImportProfileCmd extends BaseProfileCmd {
 
     @CommandLine.Option(
             names = {"-u", "--update"},
-            description = "Allow updating the profile if it already exists")
+            description =
+                    "Allow updating the profile if it already exists. Note that the configuration will be overwritten and NOT merged")
```

Comment:
what do you mean with "merged" ?

Reply:
I mean that all fields will be overwritten even if they are blank. 
If the profile has a token set and you import with the same name but with a blank string, it will override the token with the blank string

Reply:
can't we make it more "merge" like ?
it seems awkward,
it should be easy to implement, as you already have the previous version of the profile

Reply:
this is "import", you can use update for that